src/js/EditsContainer.jsx

```diff
@@ -2,29 +2,31 @@ var React = require('react');
 var request = require('superagent');
 
 var EditsSyntacticalValidity = require('./EditsSyntacticalValidity.jsx');
+var EditsMacro = require('./EditsMacro.jsx');
 var EditsHeaderDescription = require('./EditsHeaderDescription.jsx');
 
 var EditsContainer = React.createClass({
   getInitialState: function() {
     return {
-    	"syntactical": [],
-      "validity": []
+      edits: {
+        syntactical: [],
+        validity: [],
+        macro: {}
```

Comment:
Syntactical and validity are arrays because there are many different loan numbers which each have their own set of edits. For macro, however there is just one set of edits. This could be tweaked either way if we wanted everything to be either an array/object.. but I think this might be even more explicit that the data is *different*.

Additionally re: data structures, we may need to sit down and figure out all the ways we'll need to use the data so that it is stored in a format that allows the operations efficiently (if we need to look up every set of syntax edits by loan ID, perhaps rather than an arrray, we'd just store the data as an object with loanIDs as keys and edit arrays as values (or objects which contain the edit arrays and other stuff we don't yet have).